Validation/Geometry/test/runP_HCAL_Debug_cfg.py

```diff
@@ -67,6 +67,12 @@
         EtaHigh      = cms.untracked.double(3.0),
         EtaMinP      = cms.untracked.double(-5.5),
         EtaMaxP      = cms.untracked.double(5.5),
+        EtaMin0      = cms.untracked.double(2.650),
```

Comment:
May I suggest a more meaningul name for these variables? Something like etaHighMax, etaMidMax, etaLowMax,.... or at least some minimal order?

Reply:
Done